### ISSUE TITLE
KeyError: 'ws_extra_headers' in Receptor 0.6

### ISSUE THREAD

**Issue by lhellebr** [project-receptor/python-receptor#188]
Getting "KeyError: 'ws_extra_headers'" in Receptor 0.6 (shipped with latest Satellite 6.7 snap - 16). @matburt says this needs to be cherrypicked.
```
# receptor --debug -d /tmp/send send --peer=127.0.0.1:8881 node --directive=receptor_satellite:execute "$(cat data.json)"
INFO 2020-03-19 09:30:36,099  entrypoints Sending directive receptor_satellite:execute to node via 127.0.0.1:8881
DEBUG 2020-03-19 09:30:36,111  entrypoints Removing temporary directory /tmp/send/e6bd83ca-f85d-455d-9b0a-5e22d4b7f533
ERROR 2020-03-19 09:30:36,111  __main__ main: an error occured while running receptor
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/receptor/__main__.py", line 59, in main
    config.go()
  File "/usr/lib/python3.6/site-packages/receptor/config.py", line 516, in go
    self._parsed_args.func(self)
  File "/usr/lib/python3.6/site-packages/receptor/entrypoints.py", line 149, in run_as_send
    controller.run(send_entrypoint)
  File "/usr/lib/python3.6/site-packages/receptor/controller.py", line 89, in run
    self.loop.run_until_complete(app())
  File "/usr/lib64/python3.6/asyncio/base_events.py", line 484, in run_until_complete
    return future.result()
  File "/usr/lib/python3.6/site-packages/receptor/entrypoints.py", line 116, in send_entrypoint
    config.ws_extra_headers, send_message, read_responses)
  File "/usr/lib/python3.6/site-packages/receptor/config.py", line 561, in __getattr__
    value = self._config_options[key]
KeyError: 'ws_extra_headers'
```

**Comment by jhjaggars** [comment]
I actually get a different issue.

```
receptor on  release_0.6 [$?] is 📦 v0.6.1 via 🐍 v3.6.9 (receptor-7LbJpkJb-py3.6) on ☁️  us-east-1
❯ receptor --debug -d /tmp/send send --peer=127.0.0.1:8881 node --directive=receptor_satellite:execute "foo"
INFO 2020-03-19 10:18:26,678  entrypoints Sending directive receptor_satellite:execute to node via 127.0.0.1:8881
ERROR 2020-03-19 10:18:26,710  __main__ main: an error occured while running receptor
Traceback (most recent call last):
  File "/Users/jjaggars/code/receptor/receptor/entrypoints.py", line 148, in run_as_send
    controller = Controller(config)
  File "/Users/jjaggars/code/receptor/receptor/controller.py", line 19, in __init__
    self.receptor = Receptor(config)
  File "/Users/jjaggars/code/receptor/receptor/receptor.py", line 46, in __init__
    self.work_manager = (work_manager_cls or WorkManager)(self)
  File "/Users/jjaggars/code/receptor/receptor/work.py", line 20, in __init__
    work_info.info(dict(plugins=str(self.get_capabilities())))
  File "/Users/jjaggars/code/receptor/receptor/work.py", line 36, in get_capabilities
    for x in pkg_resources.iter_entry_points('receptor.worker')
  File "/Users/jjaggars/code/receptor/receptor/work.py", line 36, in <dictcomp>
    for x in pkg_resources.iter_entry_points('receptor.worker')
  File "/Users/jjaggars/Library/Caches/pypoetry/virtualenvs/receptor-7LbJpkJb-py3.6/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2352, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/Users/jjaggars/code/receptor-file/receptor_file/worker.py", line 19, in <module>
    @receptor.plugin_export(payload_type=receptor.BUFFER_PAYLOAD)
AttributeError: module 'receptor' has no attribute 'plugin_export'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/jjaggars/code/receptor/receptor/__main__.py", line 59, in main
    config.go()
  File "/Users/jjaggars/code/receptor/receptor/config.py", line 516, in go
    self._parsed_args.func(self)
  File "/Users/jjaggars/code/receptor/receptor/entrypoints.py", line 151, in run_as_send
    cleanup_tmpdir(controller)
UnboundLocalError: local variable 'controller' referenced before assignment
```

**Comment by matburt** [comment]
fixed in #189
